src/runner.py

```diff
@@ -90,7 +92,12 @@ def calculate_projected_gradients(self, base_r0: float):
         self.sensitivity_calc.params.update({"beta": beta})
 
         # Scale the eigenvalue gradient by beta to get r0_cm_grad
-        self.r0_cm_grad = beta * self.sensitivity_calc.eigen_value_gradient.eig_val_cm_grad
+        if self.method == "eig":
```

Comment:
You have to create one member variable in the sensitivity calculator, which contains the value for `self.r0_cm_grad`, thus this `if` will not be needed. In the sensitivity calculator, that member variable is filled w.r.t. the given `method`.